src/features/hud/components/ScreenshotButton.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+
+import html2canvas from "html2canvas";
+import screenshotIcon from "assets/icons/screenshot-icon.png";
+import { Modal } from "react-bootstrap";
+import { Panel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+
+export const ScreenshotButton: React.FC = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [ssImg, setSSImg] = useState("");
+
+  const handleTweetClick = () => {
+    window.open(
+      "https://twitter.com/intent/tweet?text=Checkout%20my%20Sunflower%20Land%20Farm%3A%0A%0A&ref_src=https%3A%2F%2Fsunflower-land.com&hashtags=SunflowerLand",
+      "_blank"
+    );
+    // setIsOpen(false);
+  };
+
+  const clearUrl = (url: string) => url.replace(/^data:image\/\w+;base64,/, "");
+
+  const downloadImage = (
+    content: string,
+    name: string = "My Sunflower Land Farm",
+    type: string = "jpeg"
+  ) => {
+    let link: HTMLAnchorElement = document.createElement("a");
+    link.href = `data:application/octet-stream;base64,${encodeURIComponent(
+      content
+    )}`;
+    link.download = /\.\w+/.test(name) ? name : `${name}.${type}`;
+    document.body.appendChild(link);
+    link.click();
+    document.body.removeChild(link);
+  };
+
+  const handleSaveImage = () =>
+    downloadImage(clearUrl(ssImg), "My SFL Farm", "jpeg");
+
+  function getScreenshot(): void {
+    //, {scale: 1,width: 1600,height: 1600, }
```

Comment:
commented code

Reply:
I kept them intentionally as this feat itself is in beta. 
Removed the comments. 👍